scripts/mkcloud

```diff
@@ -564,7 +564,9 @@ function setuplonelynodes()
         local mac=$(macfunc $i)
         local lonely_node
         lonely_node=$cloud-node$i
-        ${mkcloud_lib_dir}/libvirt/compute-config $cloud $i $mac 0 "$cephvolume" "$drbdvolume" $compute_node_memory $controller_node_memory $libvirt_type $vcpus $emulator $vdisk_dir "1" > /tmp/$cloud-node$i.xml
+        ${mkcloud_lib_dir}/libvirt/compute-config $cloud $i $mac 0 "$cephvolume"\
+            "$drbdvolume" $compute_node_memory $controller_node_memory\
+            $libvirt_type $vcpus $emulator $vdisk_dir "1" 1 > /tmp/$cloud-node$i.xml
```

Comment:
same here: `"1" ` is really `1` (well, is quite the opposite, but you get the idea)